shared/cfg/machines.cfg

```diff
@@ -46,3 +46,15 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
+    - s390:
+        only s390
```

Comment:
I did some try but still stuck in the `No urls provided nor any arguments produced runable tests. Please double check the executed command.`

Parameter of `only s390x` in machines.cfg should be same with which setting of avocado.
If I used `only s390`, avocado works fine. If I used `only s390x` got the no URL error.
I have read guest-os.cfg, there are not setting about arch name.(only have vm_arch_name)
I have grep all of cfg file, they looks fine....